content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -689,6 +690,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   This feature gate guards *a group* of CPUManager options whose quality level is beta.
   This feature gate will never graduate to stable.
 - `CPUManagerPolicyOptions`: Allow fine-tuning of CPUManager policies.
+- `CrossNamespaceVolumeDataSource`: Enable usage of Provision of PVCs from VolumeSnasphot in other namespaces.
```

Comment:
```suggestion
- `CrossNamespaceVolumeDataSource`: Enable provisioning a {{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}}
  (PVC) that is based on VolumeSnapshot in a different namespace from the PVC.
```

Reply:
Fixed.

Reply:
@sftim 
https://github.com/kubernetes/website/pull/37802/files#diff-f937c4715e7d2fa6db515660503a56c87c317122dc2860873c7022e20e452141R414-R415
I checked the contents of the description again and fixed.